docs/content/concepts/ops-jobs-graphs/graphs.mdx

```diff
@@ -262,6 +263,36 @@ Note that in most cases, it is usually possible to pass some data dependency. In
 
 Dagster also provides more advanced abstractions to handle dependencies and IO. If you find that you are finding it difficult to model data dependencies when using external storage, check out [IO managers](/concepts/io-management/io-managers).
 
+### Loading an asset as an input
+
+You can supply an asset as an input to one of the ops in a graph. Dagster can then use the [IO manager](/concepts/io-management/io-managers) on the asset to load the input value for the op.
```

Comment:
How do I override the IOManager used by the asset? I can do that with `@asset(Ins={"key": AssetIn(..., input_manager_key: "overriding_io_mgr")})` for assets, how do I do it with ops?

Reply:
Here are some docs: https://docs.dagster.io/concepts/io-management/unconnected-inputs#providing-an-input-manager-for-an-unconnected-input.

Reply:
Oh, I didn't put together that this is an "unconnected input" but I guess that makes sense, OK.